pydocx/export/mixins/faked_superscript_and_subscript.py

```diff
@@ -0,0 +1,68 @@
+from __future__ import (
+    absolute_import,
+    print_function,
+    unicode_literals,
+)
+
+import copy
+
+
+class FakedSuperscriptAndSubscriptExportMixin(object):
+    def parse_r_determine_applicable_styles(self, el, stack):
+        next_in_line = super(FakedSuperscriptAndSubscriptExportMixin, self)
+        styles = next_in_line.parse_r_determine_applicable_styles(el, stack)
+
+        properties = self.styles_manager.get_resolved_properties_for_element(
+            el,
+            stack,
+        )
+
+        def get_properties_with_no_font_size():
+            # Only set paragraph_properties if properties has a size.
+            if not properties.size:
+                return
+            copied_el = copy.deepcopy(el)
+            rpr = copied_el.find('./rPr')
+            if rpr is None:
+                return
+
+            size_tag = rpr.find('./sz')
+            if size_tag is None:
+                return
+
+            rpr.remove(size_tag)
+
+            return self.styles_manager.get_resolved_properties_for_element(
+                copied_el,
+                stack,
+            )
+
+        paragraph_properties = get_properties_with_no_font_size()
+
+        # If paragraph_properties is None then the size was not set
+        # (meaning it can't be bigger or smaller than the default for the
+        # paragraph, so early exit.
```

Comment:
I copied that comment from the original code and didn't notice the missing parenthesis :(. )) (there)